packages/dataclass-editor/src/components/dataclass/data/dataclass-field-utils.test.ts

```diff
@@ -0,0 +1,32 @@
+import { DataClassFieldModifier, type DataClassField } from './dataclass';
+import { isPersistent } from './dataclass-field-utils';
+
+let dataClassField: DataClassField;
+
+beforeEach(() => {
+  dataClassField = {
+    name: 'DataClassField',
+    type: 'String',
+    comment: 'comment',
+    modifiers: [],
+    annotations: []
+  };
+});
```

Comment:
I definitly think you should less use this beforeEach let construct in favor of a simple setup function.